modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ImportAndDeleteObjectEntryPerformanceTest.java

```diff
@@ -0,0 +1,197 @@
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new AssumeTestRule("assume"), new LiferayIntegrationTestRule());
+
+	public static void assume() {
+		Assume.assumeTrue(Validator.isNull(System.getenv("JENKINS_HOME")));
+	}
+
+	@Test
+	public void testImportAndDeleteObjectEntry() throws Exception {
+		_importObjectFolder();
+		_importObjectEntry();
+		_deleteObjectEntry();
+	}
+
+	private String _createObjectEntryJSON() throws Exception {
+		if (_OBJECT_ENTRY_COUNT < 1) {
+			throw new IllegalArgumentException(
+				"Must at least create 1 Object Entry!");
+		}
+
+		JSONArray jsonArray = JSONFactoryUtil.createJSONArray();
+
+		for (int i = 0; i < _OBJECT_ENTRY_COUNT; i++) {
+			jsonArray.put(JSONUtil.put("alpha", "foo"));
+		}
+
+		return jsonArray.toString();
+	}
+
+	private void _deleteObjectEntry() throws Exception {
+		try (PerformanceTimer performanceTimer = new PerformanceTimer(
+				StringBundler.concat(
+					" Delete ", _OBJECT_ENTRY_COUNT, " Object Entries"),
+				5000, null)) {
+
+			JSONArray jsonArray = _getObjectEntryIdJSONArray();
+
+			HttpInvoker httpInvoker = HttpInvoker.newHttpInvoker();
+
+			httpInvoker.body(jsonArray.toString(), "application/json");
+			httpInvoker.userNameAndPassword(_USER_NAME_AND_PASSWORD);
+			httpInvoker.httpMethod(HttpInvoker.HttpMethod.DELETE);
+			httpInvoker.path(_PATH);
+
+			httpInvoker.invoke();
+
+			long currentObjectEntryCount = _OBJECT_ENTRY_COUNT;
+
+			while (currentObjectEntryCount != 0) {
+				currentObjectEntryCount =
+					_objectEntryLocalService.getObjectEntriesCount(
+						_objectDefinition.getObjectDefinitionId());
+			}
+		}
+	}
+
+	private JSONArray _getObjectEntryIdJSONArray() throws Exception {
+		HttpInvoker httpInvoker = HttpInvoker.newHttpInvoker();
+
+		httpInvoker.userNameAndPassword(_USER_NAME_AND_PASSWORD);
+		httpInvoker.httpMethod(HttpInvoker.HttpMethod.GET);
+		httpInvoker.path(
+			"http://localhost:8080/o/c/foos/?fields=id&pageSize=" +
+				_OBJECT_ENTRY_COUNT);
+
+		HttpInvoker.HttpResponse httpResponse = httpInvoker.invoke();
+
+		JSONObject jsonObject = _jsonFactory.createJSONObject(
+			httpResponse.getContent());
+
+		return (JSONArray)jsonObject.get("items");
+	}
+
+	private void _importObjectEntry() throws Exception {
+		try (PerformanceTimer performanceTimer = new PerformanceTimer(
+				StringBundler.concat(
+					" Import ", _OBJECT_ENTRY_COUNT, " Object Entries"),
+				9000, null)) {
+
+			HttpInvoker httpInvoker = HttpInvoker.newHttpInvoker();
+
+			httpInvoker.body(_createObjectEntryJSON(), "application/json");
+			httpInvoker.userNameAndPassword(_USER_NAME_AND_PASSWORD);
+			httpInvoker.httpMethod(HttpInvoker.HttpMethod.POST);
+			httpInvoker.path(_PATH);
+
+			httpInvoker.invoke();
+
+			List<ObjectDefinition> objectDefinitions =
+				_objectDefinitionLocalService.getCustomObjectDefinitions(0);
+
+			_objectDefinition = objectDefinitions.get(0);
```

Comment:
@slnn Do not use 0 here, use Constance for more readability

Reply:
OK